### DISCUSSION TITLE
Re-ordering screen reader items by relevance to a given query

### DISCUSSION THREAD

**Discussion by almog-gueta** [nvaccess/nvda#13714] [General]
Hello, 

Currently there is an option that the screen reader will read the clickable items by alphabetical order, or by the location of the items in the website (left to right, up to down). We would like to add an option to read by relevant to a given query from user. **We are seeking for the specific script where we can add such algorithm inside NVDA?** 

Thanks in advanced!

**Comment by seanbudd** [comment]
Welcome @almog-gueta ,

Please read our [contributing wiki and guidelines](https://github.com/nvaccess/nvda/wiki/Contributing#guidelines).

Do you have plans for a UX? How would a query be entered?

Could you explain what problem this is trying to solve?

Please open a new issue and fill out the issue template fully.

Kind Regards,
NV Access Software Developers

**Comment by XLTechie** [comment]
Can you please provide an example, including a website (if applicable), that 
shows the current behavior, and then describe how you would expect that behavior 
to change?

At least as you've described it so far, I personally don't understand what 
you are getting at.
